src/lib.rs

```diff
@@ -315,7 +315,7 @@ where
             continue;
         }
         let path_str = path.to_str().unwrap();
-        if config.verbose() {
+        if config.verbose() && path_str != "stdin" {
```

Comment:
Are there other places in the code that do this check? I think there must be. In any case it would be good to factor out this condition into a function (`should_emit_verbose`, or `emit_verbose` taking a string to emit, or something like that). It would be good to use a constant rather than a magic string for `"stdin"`.